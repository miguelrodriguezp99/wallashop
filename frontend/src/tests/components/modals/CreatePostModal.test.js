import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import ModalB from  './../../../modules/app/components/modals/CreatePostModal';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

jest.mock('../../../modules/app/firebase/config.js', () => ({
}));

describe('ModalB component', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
        <MemoryRouter>
          <ModalB />
        </MemoryRouter>
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
});