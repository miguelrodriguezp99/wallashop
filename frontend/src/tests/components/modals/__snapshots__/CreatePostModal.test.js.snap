// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalB component renders correctly 1`] = `
<button
  className="z-0 group inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap font-normal subpixel-antialiased overflow-hidden tap-highlight-transparent outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 min-w-unit-20 h-unit-10 text-small gap-unit-2 [&>svg]:max-w-[theme(spacing.unit-8)] data-[pressed=true]:scale-[0.97] transition-transform-colors-opacity motion-reduce:transition-none data-[hover=true]:opacity-hover fixed bottom-10 right-10 p-5 bg-blue-500 text-white rounded-full shadow-lg"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragStart={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchCancel={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  type="button"
>
  Add Post
</button>
`;
